meta-luneos/recipes-core/packagegroups/packagegroup-luneos-extended.bb

```diff
@@ -150,7 +150,7 @@ RDEPENDS:${PN}:append:tenderloin = " alsa-utils-systemd mesa-driver-swrast rmtfs
 RDEPENDS:${PN}:append:tenderloin-halium = " ${LIBHYBRIS_RDEPENDS} ${MEMNOTIFY_RDEPENDS}"
 RDEPENDS:${PN}:append:mido = " alsa-utils-systemd mesa-driver-swrast rmtfs qrtr rpmsgexport"
 RDEPENDS:${PN}:append:mido-halium = " ${LIBHYBRIS_RDEPENDS} ${MEMNOTIFY_RDEPENDS} bluebinder"
-RDEPENDS:${PN}:append:mindphone = " ${LIBHYBRIS_RDEPENDS} ${MEMNOTIFY_RDEPENDS}"
```

Comment:
Why this change?

Reply:
Because I'm too lazy to add it to the kernel :P But it shouldn't be here I guess... I doubt we actually use memnotify currently and we need to patch it into each kernel ;)